packages/screens/Rakki/RakkiScreen.tsx

```diff
@@ -1,51 +1,22 @@
-import { useQueryClient } from "@tanstack/react-query";
-import Long from "long";
-import moment from "moment";
-import { useEffect, useState } from "react";
-import { StyleProp, TextInput, TextStyle, View, ViewStyle } from "react-native";
-
-import rakkiTicketSVG from "../../../assets/logos/rakki-ticket.svg";
-import { BrandText } from "../../components/BrandText";
-import { SVG } from "../../components/SVG";
-import { ScreenContainer } from "../../components/ScreenContainer";
-import { Box, BoxStyle } from "../../components/boxes/Box";
-import { TertiaryBox } from "../../components/boxes/TertiaryBox";
-import { PrimaryButton } from "../../components/buttons/PrimaryButton";
-import { SecondaryButton } from "../../components/buttons/SecondaryButton";
-import { GradientText } from "../../components/gradientText";
-import { UserAvatarWithFrame } from "../../components/images/AvatarWithFrame";
-import { GridList } from "../../components/layout/GridList";
-import { LoaderFullSize } from "../../components/loaders/LoaderFullScreen";
-import ModalBase from "../../components/modals/ModalBase";
-import { Username } from "../../components/user/Username";
-import { useFeedbacks } from "../../context/FeedbacksProvider";
-import { Info } from "../../contracts-clients/rakki/Rakki.types";
-import { useRakkiHistory } from "../../hooks/rakki/useRakkiHistory";
-import { useRakkiInfo } from "../../hooks/rakki/useRakkiInfo";
-import { useBalances } from "../../hooks/useBalances";
-import { useMaxResolution } from "../../hooks/useMaxResolution";
-import { useSelectedNetworkId } from "../../hooks/useSelectedNetwork";
-import useSelectedWallet from "../../hooks/useSelectedWallet";
-import { NetworkFeature, getNetworkFeature } from "../../networks";
-import { prettyPrice } from "../../utils/coins";
-import { ScreenFC } from "../../utils/navigation";
-import { errorColor } from "../../utils/style/colors";
-import {
-  fontMedium10,
-  fontSemibold12,
-  fontSemibold13,
-  fontSemibold14,
-  fontSemibold16,
-  fontSemibold28,
-} from "../../utils/style/fonts";
-import { modalMarginPadding } from "../../utils/style/modals";
-import { joinElements } from "../Multisig/components/MultisigRightSection";
-
-import { RakkiClient } from "@/contracts-clients/rakki";
-import { getKeplrSigningCosmWasmClient } from "@/networks/signer";
-
-// TODO: replace all placeholders text with real values
```

Comment:
this TODO is not addressed, we should address it or not remove it

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1434/commits/2f5bf643eded2c1093abdb65b3bcca07c06918e0#diff-8e2c5b8031f032da60995a2ec1bed47f31b3db49f0ce3de39821c1d7f26382cfR21

What are these placeholders ? I thought it was good, my bad

Reply:
at the bottom, there is hardcoded $10k and stuff like that

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1434/commits/82cad861fe5a6be2850b2b4422bf3e99f77d00cd